pkg/larker/loader/loader.go

```diff
@@ -65,11 +71,28 @@ func (loader *Loader) LoadFunc() func(thread *starlark.Thread, module string) (s
 
 		// A special case for loading Cirrus-provided builtins (e.g. load("cirrus", "fs"))
 		if module == "cirrus" {
+			starlibModules, err := loadStarlibModules()
+			if err != nil {
+				return nil, err
+			}
+
+			starlarkEnv := starlark.NewDict(len(loader.env))
+			for key, value := range loader.env {
+				if err := starlarkEnv.SetKey(starlark.String(key), starlark.String(value)); err != nil {
+					return nil, err
+				}
+			}
+
 			return starlark.StringDict{
 				"fs": &starlarkstruct.Module{
 					Name:    "fs",
 					Members: builtin.FS(loader.ctx, loader.fs),
 				},
+				"env":  starlarkEnv,
+				"http": starlibModules["http"],
+				"hash": starlibModules["hash"],
+				"json": starlibModules["json"],
+				"yaml": starlibModules["yaml"],
```

Comment:
Let's also add `zipfile`, `re` and `base64. Can do here or in a separate PR.

Reply:
See https://github.com/cirruslabs/cirrus-cli/pull/96/commits/40d9ac23705bfac15525e21c0c14fae01b6909f8.